modules/client/examples.md

```diff
@@ -2444,3 +2551,576 @@ console.log(members);
 ]
 */
 ```
+
+### Decode Add Members Action (Multisig)
+
+```ts
+import { Context, ContextPlugin, MultisigClient } from "@aragon/sdk-client";
+import { MultisigPluginSettings } from "../../src";
+import { contextParams } from "../00-client/00-context";
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+const multisigClient = new MultisigClient(contextPlugin);
+const data: Uint8Array = new Uint8Array([12, 56]);
+
+const settings: MultisigPluginSettings = multisigClient.decoding
+  .addAddressesAction(
+    data,
+  );
+
+console.log(settings);
+/*
+{
+  members: [
+    "0x12345...",
+    "0x56789...",
+    "0x13579...",
+  ],
+  minApprovals: 2,
+}
+*/
+```
+
+### Decode Remove Members Action (Multisig)
+
+```ts
+import {
+  Context,
+  ContextPlugin,
+  MultisigClient,
+  MultisigPluginSettings,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+const multisigClient = new MultisigClient(contextPlugin);
+const data: Uint8Array = new Uint8Array([12, 56]);
+
+const settings: MultisigPluginSettings = multisigClient.decoding
+  .removeAddressesAction(data);
+
+console.log(settings);
+/*
+{
+  members: [
+    "0x12345...",
+    "0x56789...",
+    "0x13579...",
+  ],
+  minApprovals: 2
+}
+*/
+```
+
+### Decode Remove Members Action (Multisig)
+
+```ts
+import { Context, ContextPlugin, MultisigClient } from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+const multisigClient = new MultisigClient(contextPlugin);
+const data: Uint8Array = new Uint8Array([12, 56]);
+
+const minApprovals: number = multisigClient.decoding
+  .updateMinApprovalsAction(data);
+
+console.log(minApprovals);
+/*
+2
+*/
+```
+
+## Multisig governance plugin client
+### Creating a DAO with a multisig plugin
+
+```ts
+import {
```

Comment:
Shouldn't this example go at the beginning, with the rest of DAO creations?

Reply:
Mmm we dont have a place to all dao creations.  Each examples folder (tokeVoting, addressListVoting, multisig) have an installation example but each insite its corresponding folder